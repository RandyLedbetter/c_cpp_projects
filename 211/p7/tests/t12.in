insert Chico Sacramento 90
insert Chico Redding 73
insert Sacramento San_Francisco 87
insert Sacramento Los_Angeles 384
distance Sacramento Los_Angeles
