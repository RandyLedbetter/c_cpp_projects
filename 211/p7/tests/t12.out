Sacramento to Los_Angeles is 384 miles
